api/helper.py

```diff
@@ -0,0 +1,14 @@
+from typing import List
+
+def split_at_pipe_with_default(string: str, list_length: int, defaults: List[str]) -> List[str]:
```

Comment:
This function look like it handles text api urls parameters? 
I think we should name it according to waht it does, in a way that makes it simple and clear to understand. The current name is perhaps too specific, to the point that its not clear where the function fits in the overall design. 

The variables inside the function can be called something more specific than "substrings" to explain what they are more clearly. 

I'm also not sure why its not grouped with the rest of the Texts API specific code? 
I like that we have a module for helper functions, but if its not reusable code, we might want to consider saving a helper module for functions such as those. 

Also see my lower comments on using class based Django views.

Reply:
i'm changing the names
it is not reusable code, for we have just one endpoint in v3 for now. when we'll have more. we'll probably use pipes.
i see no comment on using Django (catually i see just 3 comments)